Scripts/SaneDocReport/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
 - 
```

Comment:
how is this related?

Reply:
added the blank line to not mess with the next release